cli/src/semgrep/commands/ci.py

```diff
@@ -443,6 +453,18 @@ def ci(
     )
     output_handler = OutputHandler(output_settings)
 
+    per_product_excludes = {
+        product: [*exclude] if exclude else [] for product in ALL_PRODUCTS
+    }
+    excludes_from_app = (
+        {product: scan_handler.ignore_patterns for product in ALL_PRODUCTS}
```

Comment:
Seems like we're using the current ignore list response and setting it for all the products. 

Does this mean app-side i'd need to send back an empty list for `engine_config.ignores_files` once we cut over to per-product ignores?

Reply:
Actually, will this line change once `product_ignored_files` is available?

Reply:
Yeah, once we would get `product_ignored_files` that would just be used directly.